puma.gemspec

```diff
@@ -1,52 +1,19 @@
 # -*- encoding: utf-8 -*-
 
-# This is only used when puma is a git dep from Bundler, so it's a little
-# weird.
+# This is only used when puma is a git dep from Bundler, keep in sync with Rakefile
 
-d = File.read(File.expand_path("../lib/puma/const.rb", __FILE__))
-if d =~ /VERSION = "(\d+\.\d+\.\d+)"/
-  version = $1
-else
-  version = "0.0.1"
-end
+version = File.read(File.expand_path("../lib/puma/const.rb", __FILE__))[/VERSION = "(\d+\.\d+\.\d+)"/, 1] || raise
 
 Gem::Specification.new do |s|
   s.name = "puma"
   s.version = version
-
-  s.required_rubygems_version = Gem::Requirement.new(">= 0") if s.respond_to? :required_rubygems_version=
   s.authors = ["Evan Phoenix"]
-  s.date = `git log --pretty="%ai" -n 1`.split(" ").first
   s.description = "Puma is a simple, fast, threaded, and highly concurrent HTTP 1.1 server for Ruby/Rack applications. Puma is intended for use in both development and production environments. It's great for highly concurrent Ruby implementations such as Rubinius and JRuby as well as as providing process worker support to support CRuby well."
+  s.summary = "Puma is a simple, fast, threaded, and highly concurrent HTTP 1.1 server for Ruby/Rack applications"
   s.email = ["evan@phx.io"]
   s.executables = ["puma", "pumactl"]
   s.extensions = ["ext/puma_http11/extconf.rb"]
   s.files = `git ls-files`.split($/)
   s.homepage = "http://puma.io"
   s.license = "BSD-3-Clause"
-  s.rdoc_options = ["--main", "README.md"]
-  s.require_paths = ["lib"]
-  s.required_ruby_version = Gem::Requirement.new(">= 1.9.3")
```

Comment:
removing this would remove the require ruby version check from bundler AFAICT, I don't want to do that.